apps/site/assets/ts/stop/components/amenities/AmenityCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderFa from "../../../helpers/render-fa";
+
+const AmenityCard = ({
+  headerText,
+  icon
+}: {
+  headerText: string;
+  icon: JSX.Element;
+}): JSX.Element => {
+  return (
+    <div className="d-flex card justify-content-space-between">
+      <div className="m-16">
+        <div className="d-flex text-primary">
+          <div className="">{icon}</div>
+          <div className="u-bold fs-18 ps-8">{headerText}</div>
+        </div>
+        <div>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis mattis
+          faucibus erat.
```

Comment:
**suggestion**: add rendering of child components via the children prop `{children}`, then each card can populate the content like such

```jsx
<AmenityCard headerText="Accessibility" icon={icon}>
   Some accessibility <SpecificThing /><whatever />
</AmenityCard>
/* ... */
<AmenityCard headerText="Escalators" icon={otherIcon}>
   <p>Escalator things</p> {someVariable}
</AmenityCard>
```

etcetera.